docs/tutorials/projects/integrating-viam-with-openai.md

```diff
@@ -190,28 +178,24 @@ chmod +x run.sh
 
 ## Configuration
 
-Now that we've set up the rover by attaching the servo and making the tutorial software available on the Pi, we can configure the rover to:
+Now, configure your rover to:
 
 - Recognize and operate the servo
 - Make the ML detector model available for use by the Viam vision service
 
 ### 1. Configure the servo
 
-To configure your [servo](/components/servo/), go to your rover's **Config** tab and click the **Components** subtab.
+To configure your [servo](/components/servo/), go to your rover's **Config** page, scroll to the bottom and create a new instance of the `servo` component.
+Name it `servo1` (or something else if you prefer, but then you will need to update references to it in the tutorial code).
 
-Click **Create component** in the lower-left corner of the screen.
-
-Select type `servo`.
 Since you've attached your servo to a Raspberry Pi, choose the model `pi`.
-
-Name it `servo1`.
-Click **Create**.
+Click **Create Component**.
 
 {{<imgproc src="/tutorials/ai-integration/servo_component_add.png" resize="900x" declaredimensions=true alt="Adding the servo component." style="border:1px solid #000" >}}
 
-Now, in the panel for _servo1_, add the following configuration in attributes to tell `viam-server` that the servo is attached to GPIO pin 8, then press the **Save Config** button.
+Now, in the panel for `servo1`, add the following configuration in attributes to tell `viam-server` that the servo is attached to GPIO pin 8, then press the **Save Config** button.
 
-```json
+``` json
```

Comment:
all these spaces, prettier should be removing those. Are you manually adding these or are they automatically getting added?

Reply:
I don't know 😆 I don't think I manually added them, I think it was mdlint (vs code linter i have), but fixing now and will sync to get prettier up and working